transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/XmlResponseQueueTransformer.java

```diff
@@ -54,52 +366,80 @@
             LOGGER.info("Failure creating MIME type", e);
             throw new ExceptionInInitializerError(e);
         }
+
+        TYPE_NAME_LOOKUP = new ImmutableMap.Builder<AttributeType.AttributeFormat, String>()
+                .put(AttributeFormat.BINARY, "base64Binary").put(AttributeFormat.STRING, "string")
+                .put(AttributeFormat.BOOLEAN, "boolean").put(AttributeFormat.DATE, "dateTime")
+                .put(AttributeFormat.DOUBLE, "double").put(AttributeFormat.SHORT, "short")
+                .put(AttributeFormat.INTEGER, "int").put(AttributeFormat.LONG, "long")
+                .put(AttributeFormat.FLOAT, "float").put(AttributeFormat.GEOMETRY, "geometry")
+                .put(AttributeFormat.XML, "stringxml").put(AttributeFormat.OBJECT, "object")
+                .build();
+
+        String nsPrefix = "xmlns";
+
+        NAMESPACE_MAP = new ImmutableMap.Builder<String, String>()
+                .put(nsPrefix, "urn:catalog:metacard")
+                .put(nsPrefix + ":ns" + 1, "http://www.opengis.net/gml")
```

Comment:
That's not at all the issue. It's not that the namespaces duplicate. The Geometry is being marshaled into a full XML document that I'm then snipping out the relevant part of. When that parsing occurs, *sometimes* it randomly assigns the namespace prefixes in a different order. I'm frankly surprised I saw it in development, since I assume the ordering is based on a Hashmap's order and on any given JVM, objects will *usually* hash to the same order. I'm *glad* I saw the error in development, but changing these names won't help us at all. In fact...if I do it, then my regex will simply have to replace with `gml:` no matter what it was. So I'll be running the regex on every invocation instead of the occasional ones that come up with a different prefix from `ns1`.